infra/constants.py

```diff
@@ -30,9 +30,10 @@
     'python',
     'rust',
     'swift',
+    'ruby',
 ]
 LANGUAGES_WITH_COVERAGE_SUPPORT = [
-    'c', 'c++', 'go', 'jvm', 'python', 'rust', 'swift', 'javascript'
+    'c', 'c++', 'go', 'jvm', 'python', 'rust', 'swift', 'javascript', 'ruby'
```

Comment:
Do we actually have coverage support for Ruby?

Reply:
Yes. It's not perfect at the moment for pure Ruby, but it works and soon should be fixed. Works well with C extensions.

Reply:
Just to add a bit more context here: Ruby coverage support is currently okay, but not ideal. But we're also working with the upstream Ruby language maintainers to improve their stdlib coverage functionality: https://bugs.ruby-lang.org/issues/20448. This will improve Ruzzy's coverage gathering support significantly 👍